CoreFeatures/Database-postgresql/PostgresSchema/src/main/java/info/smart_tools/smartactors/database_postgresql/postgres_schema/search/Operators.java

```diff
@@ -200,7 +200,6 @@ private static void writeFieldCheckConditionWithTypeCast(
 
             query.pushParameterSetter((statement, index) -> {
                 statement.setObject(index++, queryParameterValue);
-                statement.setObject(index++, queryParameterType);
```

Comment:
Please clarify why we remove type when check condition with type cast?

Reply:
The type is not a parameter in the statement. It was set in the format string directly.